api/swagger.yaml

```diff
@@ -5663,7 +5663,7 @@ paths:
               examples:
                 example:
                   $ref: '#/components/examples/500'
-  /fabric/v4/timeServicesPackages:
+  /fabric/v4/timeServicePackages:
```

Comment:
Am I understanding correctly that these are changes that were made to an already-published spec rather than publishing a new version?  Do these changes represent mistakes that were made when writing the 4.10 spec (i.e. the spec changed after release but the API didn't) or was this an actual API change (so the API used to support `/timeServicesPackages` and the endpoint was changed to `/timeServicePackages` without a release)?

Reply:
They weren't mistakes in the 4.10 publish, they were late changes that the Fabric Architect @anchala71 wanted to include after release. I think @anchala71 would be better suited to respond to the question as he's more familiar with EPT (Equinix Precision Time) and their changes than I was for this release.

Reply:
@ctreatma , I pulled up with Avinash. These changes are a new feature that allows customers to view statistics on cloud routers. The API changes were made for 4.10 but the EPT team didn't get their API model update PR merged in time to be included. The updates were made on the backend APIs though so Avinash manually added them after. These will be the right endpoints for what is currently in production.

Reply:
Got it.  It's a bit concerning that we're changing API releases after they've been published rather than publishing a new release; I think we should dig into why that happens and what alternative approaches are available to us.

That said, within the scope of this SDK, it's a moot point, because the API and the spec are published and it's expected that the SDK will align with that.

Reply:
Totally agree. This change was surprising to me as well because I saw the PR for the update unmerged yet the SDKs were getting a new build from the same API Spec URL. I'll ask that this be avoided going forward.